ui/raidboss/data/07-dt/trial/queen-eternal-ex.txt

```diff
@@ -0,0 +1,211 @@
+# -p A04B:1000.0 -ii A008 A009 A00D A00E A00F A010 A018 A026 A027 A02F A030 A053 A054 A056
+
+# Phase one pulled from -r XwHyk2NAKnbzZLf7 -rf 3
+# Phase two pulled from -r GBaArv7yzHnXck8D -rf 17
+# A lot of manual cleanup applied
+
+# Phase one
+0.0 "--sync--" InCombat { inGameCombat: "1" } window 0,1
+12.9 "--sync--" StartsUsing { id: "A00C", source: "Queen Eternal" }
+15.6 "Aethertithe" Ability { id: "A00C", source: "Queen Eternal" } duration 27.8
+27.4 "Retribute" Ability { id: "A011", source: "Queen Eternal" }
+28.3 "Retribute" #Ability { id: "A012", source: "Queen Eternal" }
+36.4 "Retribute" Ability { id: "A011", source: "Queen Eternal" }
+37.3 "Retribute" #Ability { id: "A012", source: "Queen Eternal" }
+45.4 "Retribute" Ability { id: "A011", source: "Queen Eternal" }
+46.3 "Retribute" #Ability { id: "A012", source: "Queen Eternal" }
+58.7 "Virtual Shift" Ability { id: "A019", source: "Queen Eternal" }
+
+# Wind miniphase
+67.8 "Laws of Wind" Ability { id: "A023", source: "Queen Eternal" }
```

Comment:
```suggestion
67.8 "Laws of Wind" Ability { id: "A023", source: "Queen Eternal" } window 67.8,10
```
Feel free to ignore this as well, but when there are unique beginnings to each phase, it's a little nice to have full syncs to them for the benefit of people who might start ACT late.

Reply:
If the target is "beginning of the phase", it would instead be against this line (which is what changes the arena):

```
59.7 "Virtual Shift" Ability { id: "A019", source: "Queen Eternal" }
```

You just sort of stand around in each phase for 10+ seconds before the first castbar even starts.